flink-tests/src/test/java/org/apache/flink/test/checkpointing/RescalingITCase.java

```diff
@@ -617,7 +617,6 @@ private static JobGraph createJobGraphWithOperatorState(
         StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
         env.setParallelism(parallelism);
         env.getConfig().setMaxParallelism(maxParallelism);
-        env.enableCheckpointing(Long.MAX_VALUE);
```

Comment:
Is this relevant change?

Reply:
Yes~ Currently we would waiting for the final checkpoint if the checkpoint is enabled and skip waiting if the checkpoint is not enabled. However, if the checkpoint is enabled but the interval is set to Long.MAX_VALUE, the task would be blocked on wait for the final checkpoint, which in fact will not happen.

For the long run, perhaps we might be able to optimize this situation by skip the final checkpoint for non 2-pc operators: https://issues.apache.org/jira/browse/FLINK-25383.

Reply:
Ok, thanks for the explanation. 

Can you fill in the release notes field in the jira ticket, mentioning:

- the default configuration value has changed
- with links to the docs about the feature and how disable it
- you can refer this case as an example where this change can affect users (enabled checkpointing with long checkpointing intervals will block bounded jobs from completing until checkpoint is triggered)

Reply:
I updated the release note~